gcloud/bigtable/happybase/connection.py

```diff
@@ -349,14 +352,9 @@ def delete_table(self, name, disable=False):
                         is provided for compatibility with HappyBase, but is
                         not relevant for Cloud Bigtable since it has no concept
                         of enabled / disabled tables.
-
-        :raises: :class:`ValueError <exceptions.ValueError>`
-                 if ``disable=True``.
         """
         if disable:
-            raise ValueError('The disable argument should not be used in '
-                             'delete_table(). Cloud Bigtable has no concept '
-                             'of enabled / disabled tables.')
+            _WARN(_DISABLE_DELETE_MSG)
```

Comment:
I realized while porting the "gotchas" above that this shouldn't be raising (that was the old way).